### ISSUE TITLE
In Firefox, NVDA does not announce the focused item when navigating between options within a collapsed drop-down embedded in an iframe using up/down arrows

### ISSUE THREAD

**Issue by sjoehl** [nvaccess/nvda#9518]
### Steps to reproduce:
1. Navigate to http://ccidev.utoledo.edu/ff-nvda-bug-demo.html
2. Navigate to any of the combo boxes on the page.
3. Enter focus mode using tab or enter keys.
4. Traverse any of the combo boxes using the up/down arrows without expanding/opening the combo box.
### Actual behavior:
Nothing is announced when a new option in the combo box receives focus and the combo box is collapsed. If the combo box is expanded using space or alt+down arrow, the options are announced when traversed. If the combo box is collapsed and not embedded in an iframe, the options are announced when traversing the combo box.
### Expected behavior:
The newly focused option is announced when traversing the combo box using up/down arrow keys.
#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2018.4
#### Windows version:
Version	6.1.7601 Service Pack 1 Build 7601

#### Name and version of other software in use when reproducing the issue:
Firefox 66.0.3 64-bit
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

**Comment by DrSooom** [comment]
CC: @jcsteh and @MarcoZehe

**Comment by jcsteh** [comment]
This is a duplicate of #7819. It is due to [Mozilla bug 1473234](https://bugzilla.mozilla.org/show_bug.cgi?id=1473234), which has been fixed for Firefox 68.
